### DISCUSSION TITLE
Cloud free ipCam

### DISCUSSION THREAD

**Discussion by Cupra1979** [indomus/forum#383] [Hardware & Firmware]
Ciao a tutti!

Premessa: ho una economicissima ipCam Wansview K5 che ho di recente (l'altro ieri) installato all'esterno al coperto sotto ad un gazebo (lo so che è da interni, ma visto il costo irrisorio pagato (_tra l'altro rimborsato con un buono di pari valore dallo stesso venditore_) è sacrificabile per lo scopo al quale deve adempiere per qualche settimana.
Due giorni dopo (oggi), mi arriva una mail da Wansview dove in un italiano stentato mi chiedono come mi trovi con la mia fantastica cam e **casualmente** mi propongono la nuova ipCam da **_ESTERNO_** scontata del 50%.

_**A pensar male si fa peccato, ma spesso ci si prende (cit.)...**_

Domadona: esiste un custom firmware installabile su detta cam, per renderla indipendente da qualsiasi cloud, restando accessibile dall'esterno? Come hub uso HA su un Raspberry Pi4 in abbinamento ad Homey.
Se è una roba fattibile, esiste qualcosa per liberare in egual modo dal cloud anche una "Blink Mini" ed una "Ezviz C1C" che mi hanno regalato?
Se il gioco non vale la candela, troverò un modo dignitoso per smaltirle in modo onorevole...

Grazie mille!!

**Comment by KC-inDomus** [comment]
non fai prima a ragionare in termini dj blocco del traffico della cam verso l'esterno, usando poi i
feed video per l'interno, magari verso un NVR personale?

**Reply by Cupra1979** [reply]
Non ho idea se bloccando il traffico dati verso l'esterno con firmware originale, continuino a funzionare...
Al momento non ho NVR (ma devo installare a breve un sistema di videosorveglianza con telecamere esterne)

**Reply by KC-inDomus** [reply]
se espongono flussi interni, dovrebbero funzionare senza problemi. per NVR valuta motioneye
